anxcloud/resource_virtual_server_test.go

```diff
@@ -152,6 +152,8 @@ func TestAccAnxCloudVirtualServer(t *testing.T) {
 }
 
 func TestAccAnxCloudVirtualServerFromScratch(t *testing.T) {
+	t.Skip("specific from_scratch templates no longer available")
```

Comment:
Not sure whether we should just remove the test in this case. I don't think it'll come back and even if, we can always revert the commit for that. ^^

Reply:
There is a new more generic `from_scratch` template which is still somewhat WIP and could cause some issues that require manual intervention. I think if we remove it entirely, it will probably be overlooked in the future.

Reply:
Hm, okay, let's keep it then! :+1: